src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin product-code-font-10 {
+  font-family: $card-ff;
+  color: $dark-color;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
Mixin names should be in kebab-case. Rename `product-code-font-10` to `product-code-font-10` to comply with the code style rules.